project_generator/tools/gccarm.py

```diff
@@ -21,6 +21,7 @@
 import logging
 import ntpath
 import subprocess
+from .tool import Tool, Exporter
```

Comment:
follow this https://www.python.org/dev/peps/pep-0008/#imports, which would prevent this from happening :)